routes/articleRoutes.js

```diff
@@ -26,4 +33,13 @@ router.route('/articles')
     createArticle
   );
 
+router.route('/articles/:slug/comments')
```

Comment:
I'd like you to help take the `articles` bit of the routes in this file to `app.js` so that it looks like the others.

Reply:
Done that